### DISCUSSION TITLE
uno webassembly app - with azure communication services video calling

### DISCUSSION THREAD

**Discussion by scifi353** [unoplatform/uno#18307] [Q&A]
### Current behavior

Howdy.   
I'm trying to build an Uno WebAssembly(only) app that will contain Azure Communication Services video calling capabilities.  After creating a new Uno project in visual studio 2022 I get an error after adding the relevant nuget package.  When installing the nuget package for Azure.Communication.Calling.WindowsClient I get an error that reads the following:  

![image](https://github.com/user-attachments/assets/9f7c1f9f-ab94-4f53-83a1-0a70955c5b51)

The package is compatible with .net 8 even though the error says it's only compatible with .net 6. 
https://www.nuget.org/packages/Azure.Communication.Calling.WindowsClient

Please help, thank you.

Windows 11
Visual Studio 2022
.Net 8
Uno Platform 5.3.23.34

### Expected behavior

Desired behavior is to have Azure Communication Services video calling capabilities in Uno WebAssembly app.

### How to reproduce it (as minimally and precisely as possible)

Steps to reproduce:
1. Create new Uno WebAssembly(only) project.
2. Install nuget package:   Azure.Communication.Calling.WindowsClient

**Comment by jeromelaban** [comment]
Thanks for the question! 

As you may have noticed this package is not compatible with WebAssembly apps. It's only compatible Windows WinUI and Windows UAP projects.

In order to get azure communications services working, you'll need either an Uno compatible package, or to use a Javascript version of that SDK that can be included in [WebAssembly through JavaScript interop](https://platform.uno/docs/articles/interop/wasm-javascript-1.html).
